course5/homework12/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+project(homework12)
+cmake_minimum_required(VERSION 3.2)
+
+find_package(Boost REQUIRED COMPONENTS program_options)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -lboost_program_options")
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
This setting doesn't belong int the file.
When configuring the build from command -line, it should be the user decision which mode they want.
 Tools, like Visual Studio or QtCreator, will generate multiple configurations and allow the user the switch between them easily.